src/components/fields/submit-button/submit-button.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from "@lifesg/react-design-system";
+import React, { useEffect } from "react";
+import * as Yup from "yup";
+import { useValidationSchema } from "../../../utils/hooks";
+import { IGenericFieldProps } from "../../frontend-engine";
+import { ISubmitButtonSchema } from "./types";
+
+export const SubmitButton = React.forwardRef<HTMLButtonElement, IGenericFieldProps<ISubmitButtonSchema>>(
+	(props, ref) => {
+		// =============================================================================
+		// CONST, STATE, REF
+		// =============================================================================
+		const {
+			schema: { id, title, disabled, styleType, onClick, ...otherSchema },
```

Comment:
no need to extract disabled, styleType and onClick because they will still get passed to the button as part of otherSchema